### DISCUSSION TITLE
Vulnerability issue in dropwizard-configuration dependency : 2.1.1

### DISCUSSION THREAD

**Discussion by viksin19** [dropwizard/dropwizard#5853] [General]
After running the OWASP Dependency checker against the dropwizard, one vulnerability is raised against dropwizard-configuration dependency which is brought in: 
  snakeyaml-1.30.jar CVE-2022-25857, CVE-2022-38749, CVE-2022-38750, CVE-2022-38751, CVE-2022-38752

![image](https://user-images.githubusercontent.com/52523831/189583283-624cbf0c-5df6-43cb-aca0-38b65a4a83cc.png)

**Comment by zUniQueX** [comment]
Hi @viksin19. Dropwizard 2.1.2 upgraded to Jackson 2.13.4 (which uses snakeyaml 1.31) and was released just a few hours ago.

**Comment by brentryan** [comment]
Looks like snakeyaml 1.31 has another CVE which was fixed by snakeyaml 1.32. See [CVE-2022-38752](https://vuln.whitesourcesoftware.com/vulnerability/CVE-2022-38752)

**Reply by zUniQueX** [reply]
There is an open issue (#5870), which targets this problem.

Jackson 2.14.0-rc1 was just released. This version updated to snakeyaml 1.32, but the change isn't ported back to 2.13.x yet.
